src/nih_wayfinding/app/scripts/routing/abbreviate-filter.js

```diff
@@ -0,0 +1,105 @@
+
+(function () {
+    'use strict';
+
+    var directions = {
+        north: 'N',
+        northeast: 'NE',
+        east: 'E',
+        southeast: 'SE',
+        south: 'S',
+        southwest: 'SW',
+        west: 'W',
+        northwest: 'NW',
+        n: 'N',
```

Comment:
Okay.  Could maybe go to uppercase instead for case-normalizing then?  If there's not an easy way around that, that's fine.